apps/www/_blog/2024-01-29-ruby-on-rails-postgres.mdx

```diff
@@ -0,0 +1,105 @@
+---
+title: Getting started with Ruby on Rails and Postgres
```

Comment:
should we add "supabase" to the title? 

I'm sure that a lot of peopel will want to search for "using supabase with RoR" at some point

Reply:
what about "Getting started with Ruby on Rails and Supabase Postgres" to capture both?